### DISCUSSION TITLE
Stop auto starting servers after wipe

### DISCUSSION THREAD

**Discussion by newtarkprojects** [gameserverapp/Platform#151] [Suggestions]
Hey,

Please could I suggest that when using the Wipe all option on the dashboard, this does not auto-start servers. This causes problems when wanting to make changes to a server after a wipe, having to try to shut down all the servers waiting on the staggered approach takes some time. 

Thank you,
Gem

![image](https://user-images.githubusercontent.com/48177364/161379080-e4ab7c20-bd3a-4f32-8877-fe2b8a8d7090.png)

**Comment by mvaessen** [comment]
Thanks for your suggestions!
I've created a development item for your suggestion.
